Sources/StreamVideo/WebRTC/v2/WebRTCStateAdapter.swift

```diff
@@ -188,6 +191,11 @@ actor WebRTCStateAdapter: ObservableObject, StreamAudioSessionAdapterDelegate {
         self.incomingVideoQualitySettings = value
     }
 
+    func set(isTracingEnabled value: Bool) {
```

Comment:
When is this set?

Reply:
We get this value from JoinResponse [here](https://github.com/GetStream/stream-video-swift/blob/b2571694d248afb64798343b17fb95676f03cb35/Sources/StreamVideo/WebRTC/v2/WebRTCAuthenticator.swift#L104)